### ISSUE TITLE
Allow combinations of arrow, back arrows, and different line types

### ISSUE THREAD

**Issue by viridium** [almende/vis#351]
It would be nice to support back arrows, and combinations of back and forward arrows. It would also be nice to support various combinations of arrows and line types. This can be achieved by allowing the line type to be in its own dimension and putting the back and forward arrows in their own dimension:

Line type: solid, dashed, dotted, thick, etc.
Forward arrow: none, normal, triangle arrow, non-triangle arrow, hollow arrow, etc.
Backward arrow: like above

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
